### DISCUSSION TITLE
How to configure a single-user mode?

### DISCUSSION THREAD

**Discussion by schulle4u** [datenstrom/community#231] [Ask a question] [answered]
As I'm currently migrating a friend's website to yellow, I stumbled upon a small issue in the webinterface plugin. 
Is it possible to disable user account registration in the webinterface for sites which only need one administration account? Currently everyone knowing the edit link theoretically can at least create a pending account. I couldn't find anything in the appropriate configuration documentation. 

If this is not possible, my idea is to extend user account configuration in the way that webmaster can decide who is able to create new accounts. Either everyone (like it is currently), or only the webmaster after loggin into the interface. What do you think? 

Steffen

**Comment by markseuffert** [comment]
There is an experimental feature which you could try out. Just keep in mind that it might be changed or removed at anytime. Change the webmaster email to something with `noreply` and no user accounts  can be created in the web browser. User accounts can still be created at the command line. For example open file `system/config/config.ini` and change `Email: noreply@example.com`.

**Comment by schulle4u** [comment]
Yes, this will work as a temporary solution. However, system updates from the webinterface then will also be disabled. 

Another way could be to change `WebinterfaceLocation` into something random and remove the edit link from footer snippet. I'll have to confer with my friend and find out the best solution for her website. 

Thanks so far. 
Steffen

**Comment by markseuffert** [comment, marked as answer]
In the latest [edit extension](https://github.com/annaesvensson/yellow-edit) there's a new setting. You can disable user account creation in the web browser. Existing users can log in and change their settings, but new users can not create an new user account. Open file `system/extensions/yellow-system.ini` and change `EditLoginRestriction: 1`.

Tested with Datenstrom Yellow 0.8.17.
